src/main/java/tech/autodirect/api/database/TableUsers.java

```diff
@@ -84,4 +85,9 @@ public void removeUserByID(String userId) throws SQLException {
         // execute the above SQL statement and extract result into a Map
         stmt.executeQuery();
     }
+
+    @Override
+    public boolean checkUser(String userId) {
+        return false;
```

Comment:
We've gotta actually implement this rather than always returning false

Reply:
Name this method better, maybe checkUserExists